### ISSUE TITLE
Debian systemd service isn't enabled by default

### ISSUE THREAD

**Issue by stephenreay** [varnishcache/pkg-varnish-cache#73]
The Debian package doesn't seem to be using dh-sytemd, so the systemd service for varnish is not automatically enabled.

According to https://wiki.debian.org/Teams/pkg-systemd/Packaging this will be handled by adding `--with systemd` to the `dh` call in the `debian/rules` file

**Comment by denisbr** [comment]
I'll have to re-open this while re-working the usage of dh-systemd.
As it is #74 breaks the packaging on all non-systemd enabled distro versions, and we need to figure out the right way to handle both.

**Comment by stephenreay** [comment]
Can you define "breaks"?

**Comment by denisbr** [comment]
@stephenreay Adding in dh-systemd as a dependency makes the package job break due to missing deps on all the old distros. For some, there are dh-systemd packages available in other repos, but adding those adds a new dependency to the resulting packages that isn't resolveable on a normal install.
I've detailed the issue, and my thoughts on solving it in #80, feel free to comment / work with me on that :)

**Comment by denisbr** [comment]
During my testing I found that after implementing #74 ``varnishncsa`` would also autostart.
I've fixed this issue, and explicitly handle both services in https://github.com/varnishcache/pkg-varnish-cache/commit/65eae19a7b05cfd005beb7ed87c2b307afadc9b0 
After this, and the split out of the non-systemd packages, I think we're good. The next release of Varnish 4.1 should include the packaging fixes, so I'll close this for now.
@stephenreay thanks for your assistance on this, appreciated.
